docs/api_reference/index.rst

```diff
@@ -122,6 +122,7 @@ API Docs
    BoundedGroupLoss
    SquareLoss
    ZeroOneLoss
+   MeanLoss
```

Comment:
Looks like these were perhaps sorted alphabetically at some point and then... not?